src/api/module/VBClubMgmt/src/V1/Rest/Api/MatchEntityInterface.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace VBClubMgmt\V1\Rest\Api;
+
+interface MatchEntityInterface
+{
+    const ID = 'match_id';
+    const TOURNAMENT_ID = 'tournament_id';
+    const OPPONENT_ID = 'opponent_id';
+    const DATE = 'date';
+    const AVAILABLE_PLAYERS = 'available_players';
+    const CONFIRMED_PLAYERS = 'confirmed_players';
+    const DATA = 'data';
+    const CREATED_AT = 'created_at';
+    const UPDATED_AT = 'updated_at';
+
+    /***
+     * @return int
+     */
+    public function getMatchId(): int;
+
+    /**
+     * @param $matchId
+     *
+     * @return \VBClubMgmt\V1\Rest\Api\MatchEntityInterface
+     */
+    public function setMatchId($matchId): MatchEntityInterface;
+
+    /***
+     * @return int
+     */
+    public function getTournamentId(): int;
+
+    /**
+     * @param int $tournamentId
+     *
+     * @return \VBClubMgmt\V1\Rest\Api\MatchEntityInterface
+     */
+    public function setTournamentId($tournamentId): MatchEntityInterface;
```

Comment:
Is it possible to have instead a function 

```
setTournament(Tournament $tournament) 
```

I don't know if Apigility is able to do this BTW

Reply:
I don't know neither.
For now i'm keeping it simple and functionnal. I'll test this type of improvement in future feature.